api/tests/unit/infrastructure/utils/CampaignAssessmentCsvLine_test.js

```diff
@@ -439,7 +410,37 @@ describe('Unit | Infrastructure | Utils | CampaignAssessmentCsvLine', function()
       });
 
       context('when organization type is SUP and manage students', function() {
-        it('returns the group', function() {
+
+        context('on student number column', function() {
```

Comment:
**suggestion** : avoir le it directement ça me choque pas, je serais pour supprimer cette indentation de contexte supplémentaire

Reply:
done